src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Entity/ResetOidcSecretCommandHandler.php

```diff
@@ -65,6 +66,7 @@ public function handle(ResetOidcSecretCommand $command)
 
         if ($entity->getEnvironment() === Constants::ENVIRONMENT_PRODUCTION) {
```

Comment:
This is the same as the new $entity->isProduction() check?

Reply:
Correct!